docs/customizing_deis/choosing-a-scheduler.rst

```diff
@@ -132,9 +131,67 @@ component, or spy on Docker events directly on the swarm-manager machine:
     2015-04-30T17:31 61e59c: (from 172.17.8.100:5000/hungry-variable:v5 node:deis-02) start
 
 
+Mesos with Marathon framework
+-----------------------------
+
+.. important::
+
+    The Mesos with Marathon framework Scheduler is a technology preview and is not recommended for
+    production use.
+
+`Mesos`_ is a distributed system kernel:
+
+    Mesos provides APIs for resource management and scheduling. A framework interacts with Mesos master
+    and schedules and task. A Zookeeper cluster elects Mesos master node. Mesos slaves are installed on
+    each node and they communicate to master with available resources.
+
+`Marathon`_ is a Mesos_ framework for long running applications:
+
+    Marathon provides a Paas like feel for long running applications and features like high-availablilty, host constraints,
+    service discovery, load balancing and REST API to control your Apps.
+
+Deis uses the Marathon framework to schedule containers. Since Marathon is a framework for long-running
+jobs, Deis uses the `Fleet Scheduler`_ to run batch processing jobs. ``deisctl`` installs a standalone Mesos
+cluster. To install an HA Mesos cluster, follow the directions at `aledbf-mesos`_, and set the etcd key
+``/deis/scheduler/mesos/marathon`` to any Marathon node IP address. If a request is received by a regular
+Marathon node, it is proxied to the master Marathon node.
+
+To test the Marathon Scheduler backend, first install and start the mesos components:
+
+.. code-block:: console
+
+    $ deisctl install mesos && deisctl start mesos
+
+Then set the controller's ``schedulerModule`` to "mesos_marathon":
+
+.. code-block:: console
+
+    $ deisctl config controller set schedulerModule=mesos_marathon
+
+The Marathon framework is now active. Commands such as ``deis destroy`` or
+``deis scale web=9`` will use `Marathon`_ to manage app containers.
+
+Deis starts Marathon on port 8180. You can manage the apps through Marathon UI which is accessible at http://<Marathon-node-IP>:8180
+
+.. note::
+
+    **Known Issues**
+
+    - deisctl install a standalone mesos cluster. As deis doesn't support metadata tags for fleet.
+      keep watching `dynamic metadata fleet PR 1077`_.
+    - App containers will not be rescheduled if deis-registry is unavailable.
+    - Marathon starts on port 8180.
```

Comment:
Would you please change this to "If you want to access the Marathon UI, you'll have to expose port 8180 in the security group settings. This is blocked off by default for security purposes."